### DISCUSSION TITLE
Static route, am I doing it wrong?

### DISCUSSION THREAD

**Discussion by SpectroHA** [Ysurac/openmptcprouter#2440] [Q&A] [answered]
First thing, I was just recently recommended this project through a discussion on Reddit and it works beautifully for providing consistent uptime for me and my family, something we have not had since moving to the countryside several years ago.  My whole family loves this project.  Thank you!

I am attempting to add a static route for accessing my Starlink dish, thus:

![image](https://user-images.githubusercontent.com/10557484/181137888-fb4705f2-e05d-4fa0-9b38-344892cb9248.png)

Trace shows the route still going over the VPN:

![image](https://user-images.githubusercontent.com/10557484/181137970-231f4b9f-aadc-4fdd-bcf0-fbcf0e99f698.png)

Finally I don't see my route addition in the active routes under Status->Routes.

![image](https://user-images.githubusercontent.com/10557484/181138224-b105b855-c0a7-4fbd-b4bf-aaa7006fdd48.png)

I expect I'm missing something simple but haven't been able to figure it out.  Any help appreciated.  Thank you.

![image](https://user-images.githubusercontent.com/10557484/181138399-11554c68-0be7-4462-8b73-a91a17b98b19.png)

**Comment by SpectroHA** [comment, marked as answer]
To answer my own question I finally figured out that I needed to also add 192.168.100.1 pointing to my Starlink router in OMR-Bypass.

**Reply by Chris5342** [reply]
Please can you confirm you mean something like this on Services -> OMR-Bypass :
(I think it helps others to have a screen grab of settings) 

![Screenshot 2022-08-12 at 10 00 06](https://user-images.githubusercontent.com/105393840/184322025-e7cb4bba-9270-455f-b643-818425b26670.jpg)

It was working without this for me so not sure why you needed it.

**Reply by SpectroHA** [reply]
Not "Source Lan", the section "IPs and Networks" under "Domains"

![image](https://user-images.githubusercontent.com/10557484/185798493-d69e710c-9cb6-4f44-90a9-316559ecb3e6.png)

On my LAN I'm using pi-hole for DNS, pi-hole is then pointing to OMR for DNS.  Between OMR and my LAN is PFSense.  Perhaps one or both of those has something to do with it..
